pkg/apiserver/storage/testing/utils.go

```diff
@@ -0,0 +1,387 @@
+/*
+Copyright 2015 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testing
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"path"
+	"reflect"
+	"sync"
+	"sync/atomic"
+	"testing"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"k8s.io/apimachinery/pkg/api/meta"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/apiserver/pkg/apis/example"
+	"k8s.io/apiserver/pkg/storage"
+	"k8s.io/apiserver/pkg/storage/value"
+
+	grafanaregistry "github.com/grafana/grafana/pkg/apiserver/registry/generic"
+)
+
+var KeyFunc = func(namespace, name string) string {
```

Comment:
🎉    IIUC -- this is the "key" change between this and the original files in:
https://github.com/kubernetes/apiserver/blob/master/pkg/storage/testing/utils.go

Should we add links at the top saying this is originally from:
https://github.com/kubernetes/apiserver/blob/master/pkg/storage/testing/utils.go